src/ontology/mondo-edit.obo

```diff
@@ -463666,6 +463686,21 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 is_obsolete: true
 consider: MONDO:0000001
 
+[Term]
+id: MONDO:0700223
+name: hereditary skeletal muscle disorder
+def: "An instance of muscle tissue disorder that is caused by an inherited genomic modification in an individual." [MONDO:patterns/hereditary]
+synonym: "genetic muscle disease" EXACT [https://orcid.org/0000-0001-5208-3432]
+synonym: "genetic muscle disorder" EXACT []
+synonym: "genetic muscular disease" EXACT [https://orcid.org/0000-0001-5208-3432]
+synonym: "genetic muscular disorder" EXACT [https://orcid.org/0000-0001-5208-3432]
+synonym: "hereditary muscle disorder" EXACT [https://orcid.org/0000-0001-5208-3432]
+is_a: MONDO:0003847 {source="https://orcid.org/0000-0002-0736-9199"} ! hereditary disease
+is_a: MONDO:0020120 ! skeletal muscle disorder
```

Comment:
can you please add Peter's ORCID here too?

Reply:
done